lib/views/pages/login_signup/login_form.dart

```diff
@@ -64,20 +61,24 @@ class LoginFormState extends State<LoginForm> {
       setState(() {
         _progressBarState = false;
       });
-      _exceptionToast(
-          'Connection Error. Make sure your Internet connection is stable');
+      Fluttertoast.showToast(
+          msg: 'Connection Error. Make sure your Internet connection is stable',
+          backgroundColor: UIData.toastErrorColor);
     } else if (result.hasException) {
       print(result.exception);
       setState(() {
         _progressBarState = false;
       });
 
-      _exceptionToast(result.exception.toString().substring(16, 35));
+      Fluttertoast.showToast(
+          msg: result.exception.toString().substring(16, 35),
+          backgroundColor: UIData.toastErrorColor);
     } else if (!result.hasException && !result.loading) {
       setState(() {
         _progressBarState = true;
       });
-      _successToast("All Set!");
+      Fluttertoast.showToast(
```

Comment:
Separate Toast one for success `Fluttertoast.showSuccessToast`  and one for exception `Fluttertoast.showExceptionToast` will be great. It can help in optimising UI of toast accordingly and easily plus it will also make code more understandable to developers.

Reply:
@sumitra19jha  Done.